srcpkgs/unrar/template

```diff
@@ -11,14 +11,35 @@ distfiles="https://www.rarlab.com/rar/unrarsrc-${version}.tar.gz"
 checksum=d05022442009202a792e588bec58921c123ff046fc755f7f2272871a5bd79636
 repository=nonfree
 
+post_extract() {
+	cp -a . ../libunrar
+}
+
 do_build() {
 	vsed -e 's/^\(CXXFLAGS\)=\(.*\)/\1+=\2/' \
 		 -e 's/^\(LDFLAGS\)=\(.*\)/\1+=-lpthread \2/' \
 		 -i makefile
 	make CXX="$CXX" LD="$LD" STRIP=: -f makefile
+	make CXX="$CXX" LD="$LD" STRIP=: -C ../libunrar -f makefile lib
```

Comment:
```suggestion
	make CXX="$CXX" LD="$LD" STRIP=: -f makefile lib
```
Then we should be able to just vinstall from `$wrksrc`. We won't need `cp -a . ../libunrar` in post_install which is a bit problematic because `./xbps-src clean unrar` won't remove it.

Reply:
ye I would like that, but the issue is that the make is destructive so can't build both the lib and the non-lib, I can copy the libunrar build folder inside the unrar wrksrc however, that should resolve your issue?

Reply:
hmm, I now see why you did it this way, but this could be better:
```sh
do_build() {
	vsed -e 's/^\(CXXFLAGS\)=\(.*\)/\1+=\2/' \
		 -e 's/^\(LDFLAGS\)=\(.*\)/\1+=-lpthread \2/' \
		 -i makefile
	make CXX="$CXX" LD="$LD" STRIP=: -f makefile
	vbin unrar
	make CXX="$CXX" LD="$LD" STRIP=: -f makefile lib
}
do_install() {
	vlicense license.txt LICENSE
}
```

Reply:
install in the build :scream:

Reply:
Well, each makefile target depends on the clean target, and patching it out breaks the second build... If you want to fix it, go ahead, but I think this is much better than a second copy of the source.

Reply:
well, it doesn't work either :/ the libunrar.so is never made

Reply:
scratch that I am a dumbass and didn't update the vinstall lol